src/components/QuicksightDashboardPage.tsx

```diff
@@ -0,0 +1,56 @@
+import * as React from "react";
+import { Store } from "redux";
+import { RootState } from "../store";
+import * as PropTypes from "prop-types";
+import Header from "./Header";
+import Footer from "./Footer";
+import title from "../utils/title";
+import QuicksightDashboard from "./QuicksightDashboard";
+
+export interface QuicksightDashboardPageProps extends React.Props<QuicksightDashboardPageProps> {
+  params: {
+    library?: string;
+  };
+}
+
+export interface QuicksightDashboardPageContext {
+  editorStore: Store<RootState>;
+}
+
+/** Page holds quicksight dashboards. */
+export default class DashboardPage extends React.Component<QuicksightDashboardPageProps> {
```

Comment:
We should use `QuicksightDashboardPage` -- rather than `DashboardPage` -- here, since it could be confusing. `DashboardPage` is used in the `DashboardPage` component, so using it here could be confusing.

Reply:
Good point I'll fix that.